lib/animas/animasSimulator.js

```diff
@@ -74,6 +74,21 @@ exports.make = function(config){
     return e;
   }
 
+  function validateDuration(basal) {
```

Comment:
Fair point. I wasn't too happy with the name either but couldn't think of something better - `truncateDuration` works for me too.